resources/assets/js/volumes/cloneForm.vue

```diff
@@ -70,11 +70,11 @@ export default {
         getCloneUrl() {
             return this.cloneUrlTemplate.replace(':pid', this.selectedProjectId);
         },
-    },
-    methods: {
-        setDefaultProject() {
+        defaultProjectName() {
```

Comment:
This is still more like the `selectedProjectName` rather than the `defaultProjectName` if I'm not mistaken.